src/backend/devices/s300/pyusbtut.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
think this also should be under bench folder

Reply:
Don't worry too much about this, just me learning how to use the library. If you think it's useful we can move it to the bench.

Reply:
we can have a look at it but in another PR